apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -72,6 +70,57 @@ class Comment extends Component {
     return <div style={styles.error}>{javalabMsg.commentUpdateError()}</div>;
   };
 
+  toggleHideResolved = () => {
+    this.setState({hideWhileResolved: !this.state.hideWhileResolved});
+  };
+
+  getListItems = () => {
+    const {
+      viewAsCodeReviewer,
+      viewAsTeacher,
+      onDelete,
+      onResolveStateToggle
+    } = this.props;
+    const {isResolved} = this.props.comment;
+    const {hideWhileResolved} = this.state;
+    let listItems = [];
```

Comment:
nit: this could potentially be a bit more descriptive, e.g. `options` or `menuItems`, something like that?

Reply:
Good call. Thanks!